python/taichi/main.py

```diff
@@ -438,6 +472,9 @@ def main(debug=False):
         fn = f'taichi-src-v{ver[0]}-{ver[1]}-{ver[2]}-{commit}-{md5}.zip'
         import shutil
         shutil.move('release.zip', fn)
+    elif mode == "example":
+        example = sys.argv[2:]
+        run_example(name=example)
```

Comment:
I'm slightly confused by letting `example` be a `list` here. How about `assert(len(sys.argv) == 3)` and using `example = sys.argv[2]`?

Reply:
Sorry for the confusion! I was struggling with this a bit too, the rationale was:

1. if we handle the invalid arguments here prior to the `run_example()` function call it could fail earlier and doesn't have to pass a confusing list. But we lose the chance to load available examples list/set and print them to the user.
2. passing in a list will give us better helper messages but as you mentioned, brings confusion here. 

I think if this is confusing even to reviewers, I will roll back to `1.` for simplicity!